### PR TITLE
Issue fixed for #57133

### PR DESCRIPTION
**Pull request by monzuralam**
Trac ticket: [[57133]](https://core.trac.wordpress.org/ticket/57133)

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by rudlinkon** [comment]
Thanks to your PR you can escape the $label variable also.

File: src/wp-admin/user-new.php [hunk 1]
```diff
@@ -443,7 +443,7 @@
 <table class="form-table" role="presentation">
 	<tr class="form-field form-required">
 		<th scope="row"><label for="adduser-email"><?php echo $label; ?></label></th>
```
Comment (hz-tyfoon) [inline src/wp-admin/user-new.php]: Hey @monzuralam, your PR looks good to me. 
Also, I think U can use `esc_html` for the `$label` just one line above ( line 445 )

Comment (monzuralam) [reply src/wp-admin/user-new.php]: Hey @hz-tyfoon, Thanks bhaia.
The label escaping issue has been fixed.

**Review by hz-tyfoon — APPROVED** [review]
Thanks. Looking pretty good now.

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in [r54857](https://core.trac.wordpress.org/changeset/54857).

**Pull request closed without merging**